eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/review/review.controller.js

```diff
@@ -0,0 +1,164 @@
+﻿(function(angular) {
+angular.module('equizModule').controller('ReviewController', ReviewController);

```

Comment:
Use the following convention:

angular
.module('equizModule')
.controller('ReviewController', ReviewController);